### ISSUE TITLE
2025-11-06: vllm-project/vllm 仓库更新报告

### ISSUE THREAD

**Issue by yan97ao** [yan97ao/argus#397]
# 每日更新报告（2025-11-06）

## vllm-project/vllm

| 提交时间 | 作者 | 提交信息 |
|----------|------|----------|
| 2025-11-06 23:47:11 | StanHatko | [HARDWARE][CPU] Add Option for Disabling Binding to Specific CPU Cores (#27953) |
| 2025-11-06 23:30:26 | Milos Puzovic | [Doc] Add Arm CPUs are on the list of supported targets in vLLM (#26018) |
| 2025-11-06 23:29:46 | Eric Yue | [Kernel][Model] Tune fused_moe Triton configs for MiniMax-M2 on H100 (#28200) |
| 2025-11-06 23:06:33 | Harry Mellor | [Docs] Switch to directory style URLs (#28058) |
| 2025-11-06 22:19:07 | Michael Goin | Disable nm-testing models with issues in CI (#28206) |
| 2025-11-06 19:04:18 | xiangze-arm | [CPU]Improve cpu fused moe perf (#27244) |
| 2025-11-06 18:40:03 | Chauncey | [Frontend] OpenAI Responses API supports Tool/Function calling - non-harmony  (#26874) |
| 2025-11-06 17:52:13 | courage17340 | [Bugfix][Kernel] fix merge attn states when both prefix and suffix are empty (#28181) |
| 2025-11-06 16:05:32 | wangxiyuan | [V0 deprecation]clean up is_v1_supported_oracle (#28116) |
| 2025-11-06 15:32:05 | Aditya Tewari | [CPU] Enable torch profiling (#28130) |
| 2025-11-06 15:07:36 | Seungduk Kim | Fix hard-coded parameter name in gemma3n.py (#27946) |
| 2025-11-06 14:43:16 | Julien Denize | Patch Mistral Tokenizer (#28146) |
| 2025-11-06 13:52:16 | Xiaozhu Meng | [flashinfer] fix FI all2all with FI cutlass moe (#28166) |
| 2025-11-06 13:41:47 | gmagogsfm | [CI] Add compile/test_multimodal_compile.py to CI (#28151) |
| 2025-11-06 13:08:55 | Jacob Zhong | Make the cv2 dependency optional (#27780) |
| 2025-11-06 12:05:12 | Lukas Geiger | [Core][MM] Use non-blocking CPU-GPU copy of multimodal data (#28141) |
| 2025-11-06 11:46:30 | Isotr0py | [Chore] Clean up deepseek v2/v3 config copy (#28055) |
| 2025-11-06 10:07:47 | Chauncey | [Misc] Remove the duplicate code (#28111) |
| 2025-11-06 09:22:13 | Vadim Gimpelson | [CI Failure] `nm-testing/Qwen2-0.5B-Instruct-FP8-SkipQKV` was removed from HF. Skip it in tests (#28170) |
| 2025-11-06 09:21:08 | Wentao Ye | [Feature] Enable TP + EP `shared_experts` overlap with router, 3.7% E2E performance improvement (#28164) |
| 2025-11-06 09:04:20 | Wentao Ye | [Bug] Fix env string `"0"` same to `True` (#28159) |
| 2025-11-06 09:03:09 | Wentao Ye | [Bug] Fix cpu disable shared_experts `VLLM_DISABLE_SHARED_EXPERTS_STREAM` (#28157) |
| 2025-11-06 09:01:12 | Vadim Gimpelson | [PERF] Decouple projections from GDN custom op. Attempt 2 (#28083) |
| 2025-11-06 08:52:52 | Dayeol Lee | [Debugging] Add annotation for easier trace analysis (#22496) |
| 2025-11-06 08:12:00 | Kuntai Du | [Core][Hybrid allocator + connector 2/n] Unify `remove_skipped_blocks` by `get_last_useful_token` (#25431) |
| 2025-11-06 07:40:24 | Zhewen Li | [CI/Build] Fix `test_defaults_with_usage_context` in AMD CI (#27926) |
| 2025-11-06 07:40:10 | Zhewen Li | [CI/Build] Update checking logic in cutlass_group_gemm_supported  (#27948) |
| 2025-11-06 05:45:29 | Snehlata | [Feature]: Add corrupted request metric to V1 metrics system. (#27306) |
| 2025-11-06 05:31:46 | Richard Zou | [Docs] Add guide to debugging vLLM-torch.compile integration (#28094) |
| 2025-11-06 04:06:45 | Isotr0py | [Chore] Remove Nemotron-Nano-VL config copy (#28126) |
| 2025-11-06 03:25:44 | R3hankhan | [Hardware][IBM Z] Optimize s390x Dockerfile (#28023) |
| 2025-11-06 03:01:34 | Michael Yao | [Docs] Clean up README_TUNING.md (#28088) |
| 2025-11-06 02:33:50 | wang.yuqi | [Bugfix] Fix Qwen3-Reranker-8B load (#28117) |
| 2025-11-06 02:04:49 | Paul Zhang | [Feature] Extend batch invariant torch.compile to B200 (#27856) |
| 2025-11-06 01:45:57 | Samuel Shen | [CI]: Add LMCacheConnector Unit Tests (#27852) |
| 2025-11-06 01:25:32 | Chen Zhang | [FlashInfer] Avoid FlashInfer block_size 16 + head_size 256 on blackwell (#27994) |
| 2025-11-06 01:22:44 | gmagogsfm | [Bugfix] vLLM should check Inductor config for compile cache enablement status (#27637) |
| 2025-11-06 01:15:06 | Walter Beller-Morales | [Core] add support for reasoning parser plugins (#28075) |
| 2025-11-06 01:14:55 | Jiangyun Zhu | [Kernel] Fuse computation of g and beta for Gated Delta Net (#28095) |
| 2025-11-06 01:12:59 | Jiaju Zhang | [misc] add vLLM Beijing Meetup (#28127) |
| 2025-11-06 00:53:33 | Isotr0py | [Bugfix] Validate custom logits processor xargs for online serving (#27560) |
